source/Main.hx

```diff
@@ -38,6 +39,7 @@ class Main extends Sprite
     // We need to make the crash handler LITERALLY FIRST so nothing EVER gets past it.
     CrashHandler.initialize();
     CrashHandler.queryStatus();
+    WindowUtil.setWindowTitle("Friday Night Funkin'");
```

Comment:
I don't think this is needed as this only ever runs once and game already sets the title to that by default.

Reply:
ah

Reply:
is it like 
`Application.current.window.title = "Friday Night Funkin'";`
or they make a `public function setWindowTitle(titleWindow:String){Application.current.window.title = titleWindow;}` on class WindowUtil
well cool a idea !!!